discovery_child_development/pipeline/openalex/binary_classifer/03_binary_classifier_training_data.py

```diff
@@ -0,0 +1,126 @@
+"""
+Prepare labelled data for training a classifier
+--------------
+
+For the existing dataset of OpenAlex docs (already preprocessed with 01_preprocess_openalex_broad.py and 01_preprocess_openalex.py)
+* create a test set of 1000 docs (500 relevant, 500 not relevant) for testing the classifier after validation
+* create 3 training/validation sets for the classifier:
+    1. 50% of the data is from the EY seed list, 50% is from the broader concepts (relavant/non-relevant)
+    2. 20% of the data is from the EY seed list, 80% is from the broader concepts (relavant/non-relevant)
+    3. All of the data is from the EY seed list and broader concepts (relavant - ~11% /non-relevant - ~89%)
+* save the training/validation/test sets to S3
+
+Usage:
+
+python discovery_child_development/pipeline/binary_classifier/03_binary_classifier_training_data.py
+"""
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from nesta_ds_utils.loading_saving import S3
+from discovery_child_development import logging, config, S3_BUCKET
+from discovery_child_development.getters import openalex, openalex_broad_concepts
+
+CONCEPT_IDS = "|".join(config["openalex_concepts"])
+BROAD_CONCEPT_IDS = "|".join(config["openalex_broad_concepts"])
+YEARS = [str(y) for y in config["openalex_years"]]
+YEARS = "-".join(YEARS)
+
+OUT_PATH = "data/openAlex/processed/binary_classifier/"
+
+# needed for train-validation split
+SEED = config["seed"]
+
+if __name__ == "__main__":
+    # Loading in the data
+    logging.info("Loading in the data...")
+    openalex_data = openalex.get_abstracts()
+    openalex_broad_data = openalex_broad_concepts.get_abstracts_broad()
+
+    # Create test set for the classifier
+    logging.info("Creating test set...")
+    # 50/50 split of EY seed list and broader concepts
+    test_data = pd.concat(
+        [openalex_broad_data.sample(500), openalex_data.sample(500)],
+        keys=["not relevant", "relevant"],
```

Comment:
FYI I think using `keys` sets the label to be the index, and as a result the getter for the test set brings back a dataframe with an extra column `Unnamed: 1`